app/src/main/java/com/douglasbruce/tasky/core/data/repository/EventRepositoryImpl.kt

```diff
@@ -62,6 +64,16 @@ class EventRepositoryImpl @Inject constructor(
             }
             AuthResult.Success(Unit)
         }
+
+        return if (result is AuthResult.Error) {
+            dao.upsertModifiedEvent(
+                ModifiedEventEntity(
+                    eventId = event.id,
+                    type = ModificationType.Created
+                )
+            )
```

Comment:
Here, you'd also need to take care of cancellation, otherwise this will never be synced